core/src/main/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBase.scala

```diff
@@ -376,15 +390,17 @@ trait DataFrameSuiteBaseLike extends SparkContextProvider
     rdd.zipWithIndex().map{ case (row, idx) => (idx, row) }
   }
 
-  def approxEquals(r1: Row, r2: Row, tol: Double): Boolean = {
-    DataFrameSuiteBase.approxEquals(r1, r2, tol)
+  def approxEquals(r1: Row, r2: Row, tol: Double,
+                   tolTimestamp: Duration): Boolean = {
+    DataFrameSuiteBase.approxEquals(r1, r2, tol, tolTimestamp)
   }
 }
 
 object DataFrameSuiteBase {
 
   /** Approximate equality, based on equals from [[Row]] */
-  def approxEquals(r1: Row, r2: Row, tol: Double): Boolean = {
+  def approxEquals(r1: Row, r2: Row, tol: Double,
+                   tolTimestamp: Duration): Boolean = {
```

Comment:
It might be good to have a default value for tolTimestamp just incase anyone was using this API directly but optional/minor.

Reply:
I created new methods so it won't break default behaviour:
* `def approxEquals(r1: Row, r2: Row, tol: Double): Boolean` 🆕
* `def approxEquals(r1: Row, r2: Row, tolTimestamp: Duration): Boolean` 🆕
* `def approxEquals(r1: Row, r2: Row, tol: Double, tolTimestamp: Duration): Boolean`

check it out and let me know if this solves the issue

Reply:
Awesome, love it :)